app/models/manageiq/providers/openstack/storage_manager/cinder_manager/cloud_volume.rb

```diff
@@ -87,7 +87,8 @@ def self.params_for_create(ems)
   def self.raw_create_volume(ext_management_system, options)
     options = options.symbolize_keys
 
-    cloud_tenant = options.delete(:cloud_tenant)
+    cloud_tenant_id = options.delete(:cloud_tenant_id)
+    cloud_tenant    = CloudTenant.find_by(:id => cloud_tenant_id) if cloud_tenant_id
```

Comment:
Does this need RBAC? (i.e. should this be `resource_search` instead?)

Reply:
We should definitely be doing that on the API side before this is queued (would need to check to see if it is), but this is on the generic worker side of the queue we don't have access to resource_search

Reply:
Ah ok...this is fine then